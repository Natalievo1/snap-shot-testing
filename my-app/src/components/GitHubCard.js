import React from 'react';
import Card from 'react-bootstrap/Card';

const GitHubCard = () => (
  <Card style={{ width: '18rem' }}>
    <Card.Img variant="top" src="../assets/natalie.jpg" />
    <Card.Body>
      <Card.Title>Natalie Ojeda</Card.Title>
      <Card.Text>
        I live in California.
      </Card.Text>
    </Card.Body>
  </Card>
);

export default GitHubCard;
