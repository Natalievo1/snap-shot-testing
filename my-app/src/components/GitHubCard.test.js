import React from 'react';
import { render } from '@testing-library/react';
import GitHubCard from './GitHubCard';

it('GitHubCard renders correctly', () => {
  const { asFragment } = render(<GitHubCard />);
  expect(asFragment()).toMatchSnapshot();
});
