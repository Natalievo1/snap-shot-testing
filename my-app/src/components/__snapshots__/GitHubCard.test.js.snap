// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitHubCard renders correctly 1`] = `
<DocumentFragment>
  <div
    class="card"
    style="width: 18rem;"
  >
    <img
      class="card-img-top"
      src="../assets/natalie.jpg"
    />
    <div
      class="card-body"
    >
      <div
        class="card-title h5"
      >
        Natalie Ojeda
      </div>
      <p
        class="card-text"
      >
        I live in California.
      </p>
    </div>
  </div>
</DocumentFragment>
`;
